src/action/drop.rs

```diff
@@ -33,8 +33,7 @@ impl crate::sync::Database {
 }
 
 /// Drops the database, deleting all data, collections, and indexes stored in it.  Create by calling
-/// [`Database::drop`] and execute with `await` (or [`run`](DropDatabase::run) if using the sync
-/// client).
+/// [`Database::drop`].
```

Comment:
Now that the `Action` trait documents this, it doesn't need to be on every struct.